cves/CVE-2002-0392.yml

```diff
@@ -3,21 +3,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 20
```

Comment:
Could this also be a sort of arbitrary code execution CWE?  Seems it could maybe be more specific than just lack of input validation

Reply:
So you're saying that while the cause is a lack of input validation, the symptom is a denial-of-service or arbitrary code execution vulnerability, so one of those should be the real CWE? After a brief search I couldn't find any CWEs that fit the bill, but I'll keep looking.

Reply:
CWE-94 may apply, but its a bit of a stretch.

Reply:
Yeah I didn't have any one in mind, just thought maybe there would be something more specific, but if not, I think this is still accurate

Reply:
Going to just leave it as CVE-20. I wasn't able to find anything I thought was more applicable.